tests_new/integration_tests/modules/worksheets/test_worksheet.py

```diff
@@ -0,0 +1,62 @@
+from assertpy import assert_that
+
+from integration_tests.modules.worksheets.queries import (
+    create_worksheet,
+    delete_worksheet,
+    get_worksheet,
+    list_worksheets,
+    run_athena_sql_query,
+    update_worksheet,
+)
+from integration_tests.errors import GqlError
+
+
+def test_create_worksheet(client1, worksheet1):
+    assert_that(worksheet1.worksheetUri).is_length(8)
+    assert_that(worksheet1.label).is_equal_to('worksheet1')
+
+
+def test_delete_worksheet(client1, group1, session_id):
+    ws = create_worksheet(client1, 'worksheetdelete', group1, tags=[session_id])
+    assert assert_that(ws.label).is_equal_to('worksheetdelete')
```

Comment:
* double assert
* also it might be nicer to `assert_that(ws).contains_entry(label='worksheetdelete')` because if it fails it will print a message and will show the entire payload of `ws` which might be useful for debugging